presto-docs/src/main/sphinx/functions/tdigest.rst

```diff
@@ -0,0 +1,41 @@
+=========================
+T-Digest Functions
+=========================
+
```

Comment:
Add a introductory sentence

Reply:
Could you please help me with that? Isn't the data structure description enough for an introduction?